README.md

```diff
@@ -283,22 +209,18 @@ pip install .
   * [x] `regtok update <registration token>`
   * [x] `regtok delete <registration token>`
```

Comment:
I suggest you kick out the entire list here. Sooner or later and not defined in what form exactly, I suggest here we just link to the Sphinx docs page that holds a feature list / implementation status list